src/components/shared/formItems/ProjectReserved.tsx

```diff
@@ -66,9 +93,15 @@ export default function ProjectReserved({
           sliderValue={value}
           defaultValue={value ?? 0}
           suffix="%"
-          onChange={onChange}
+          onChange={value => {
+            setShowRiskWarning(
```

Comment:
**question** do we need a state variable for this? Could we instead compute `showRiskWarning` from the `value` prop?

`const showRiskWarning = (value ?? 0) > RESERVED_RATE_WARNING_THRESHOLD_PERCENT;`

Reply:
The `value` prop doesn't update `onChange`, so I think necessary